### DISCUSSION TITLE
Unable to update Ente photos thru Obtanium

### DISCUSSION THREAD

**Discussion by reachvineetjain** [ente-io/ente#4519] [General]
### Description

I am unable to update to a newer version of the application. I have installed the app thru obtanium instead of fdroid.
![Screenshot_20241226_112724_Obtainium](https://github.com/user-attachments/assets/7b19d3c7-8497-4275-be12-4bc30ef0b7ed)
It just gives me a package  ID error.

### Version

0.9.66

### What product are you using?

Ente Photos

### What platform are you using?

Mobile - Android

**Comment by afermar** [comment]
I had the same problem. It happens because Auth and Photos releases are published in the same repo. In the app additional settings in Obtainium you can specify "photos" in the filter by title field. This solved the problem for me.
